app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,72 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[show edit update destroy]
+
+  # GET /foods or /foods.json
+  def index
+    @foods = Food.includes(:user).all
+  end
+
+  # GET /foods/1 or /foods/1.json
+  def show; end
```

Comment:
Just as the previous reviewer mentioned, it is still impossible to add an ingredient😉. it generates the error below.

<img width="1080" alt="image" src="https://github.com/Lornakaboro/Recipe-app/assets/64643995/1606b529-9a8b-4c01-bd56-4c84721d7143">